nncf/common/pruning/utils.py

```diff
@@ -239,14 +239,47 @@ def get_cluster_next_nodes(graph: NNCFGraph, pruned_groups_info,
     return next_nodes
 
 
-def count_flops_for_nodes(graph: NNCFGraph,
-                          input_shapes: Dict[str, tuple], output_shapes: Dict[str, tuple],
-                          conv_op_metatypes: List[Type[OperatorMetatype]],
-                          linear_op_metatypes: List[Type[OperatorMetatype]],
-                          input_channels: Dict[str, int] = None,
-                          output_channels: Dict[str, int] = None) -> Dict[str, float]:
+def count_flops_and_weights(graph: NNCFGraph,
+                            input_shapes: Dict[str, tuple],
+                            output_shapes: Dict[str, tuple],
+                            conv_op_metatypes: List[Type[OperatorMetatype]],
+                            linear_op_metatypes: List[Type[OperatorMetatype]],
+                            input_channels: Dict[str, int] = None,
+                            output_channels: Dict[str, int] = None) -> Tuple[int, int]:
     """
-    Counts the number FLOPs in the model for convolution and fully connected layers.
+    Counts the number weights and FLOPs in the model for convolution and fully connected layers.
+
+    :param graph: NNCFGraph.
+    :param input_shapes: Dictionary of input dimension shapes for convolutions and
+                         fully connected layers. E.g {node_name: (height, width)}
```

Comment:
Please. use 4 spaces
```suggestion
    :param input_shapes: Dictionary of input dimension shapes for convolutions and
         fully connected layers. E.g {node_name: (height, width)}
```

Reply:
Done